### PR TITLE
Improve amplicon report

### PR DESCRIPTION
**Pull request by jkokosar**
Resolves https://github.com/genialis/resolwe-bio/issues/228

### CODE REVIEW

File: resolwe_bio/tools/make_report.py [hunk 1]
```diff
@@ -66,6 +128,33 @@ def parse_targetPCRmetrics(metrics_report):
...
+def get_mean_coverage(covd):
+    """Calculate mean coverage from coverageBed (.covd) report file."""
+    with open(covd) as pbc:
+        cov = 0
+        lines = 0
+        for line in pbc:
+            cov = cov + int(line.strip().split('\t')[6])
+            lines = lines + 1
+
+    return float(cov) / lines
```
Comment (dblenkus) [inline resolwe_bio/tools/make_report.py]: Use `len(pbc)` instead of `lines`.

File: resolwe_bio/tools/make_report.py [hunk 2]
```diff
@@ -66,6 +128,33 @@ def parse_targetPCRmetrics(metrics_report):
         return dict(zip(labels, values))
 
 
+def get_mean_coverage(covd):
+    """Calculate mean coverage from coverageBed (.covd) report file."""
+    with open(covd) as pbc:
+        cov = 0
```
Comment (dblenkus) [inline resolwe_bio/tools/make_report.py]: `cov = sum(int(line.strip().split('\t')[6]) for line in pbc)`

File: resolwe_bio/tools/make_report.py (hunk 1 above)
Comment (dblenkus) [inline resolwe_bio/tools/make_report.py]: You don't need to cast ints to floats, if you add `from __future__ import division`, as this is default behaviour in python 3. Transition to python 3 will also be easier, if you use `future` features.

File: resolwe_bio/tools/make_report.py [hunk 3]
```diff
@@ -117,33 +193,64 @@ def make_scatterplot(x, y, hline=None, savefile=None, dpi=300):
...
-        cov_list, _ = parse_tsv_to_list(args.cov)
-        amplicons = [line[4] for line in cov_list]
-        x = list(range(len(amplicons)))
-
-        # Barplot:
-        cov1 = [float(line[8]) for line in cov_list]
-        make_barplot(x, cov1, savefile='barplot.png')
-        template = template.replace('{#IMAGE1#}', 'barplot.png')
+        cov_list, _ = _tsv_to_list(args.cov)
+        x = list(range(len(cov_list)))
```
Comment (dblenkus) [inline resolwe_bio/tools/make_report.py]: Use more descriptive name

File: resolwe_bio/processes/exome_coverage/coveragebed.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+# Bedtools coverage (coveragebed)
+# ===============================
+---
+
+- slug: coveragebed
+  name: coverageBed
+  requirements:
+    expression-engine: jinja
+  data_name: "Coverage ({{ alignment|sample_name|default('?') }})"
+  version: 2.0.7
```
Comment (jkokosar) [inline resolwe_bio/processes/exome_coverage/coveragebed.yml]: Also, process description is fixed!

**Pull request merged**